src/utility/translations.rs

```diff
@@ -584,6 +638,7 @@ pub fn deep_sea_translation(language: Language) -> &'static str {
         Language::ES => "Para sumergirse en el tráfico de la red",
         Language::PL => "Aby zanurzyć się w ruchu sieciowym",
         Language::DE => "Um in den Netzwerkverkehr einzutauchen",
+        Language::UA => "Оригінальний, темний мотив Sniffnet",
```

Comment:
I was rapidly looking at the translations and found out that this one I equal to one of the preceding and I think it's not in the correct spot

Reply:
Good catch, will fix

Reply:
@GyulyVGC  done